app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,52 @@
+class PostsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[index show]
```

Comment:
- [ ] The Post controller uses `before_action` to restrict access to the `new` and `create` methods.